### PR TITLE
Dispatch an event when toggling dark mode

### PR DESCRIPTION
**Pull request by mohamedsabil83**

### CODE REVIEW

File: packages/admin/resources/views/components/layouts/app/topbar/user-menu.blade.php [hunk 1]
```diff
@@ -35,8 +35,10 @@
 
                 if (this.theme === 'dark' && (! document.documentElement.classList.contains('dark'))) {
                     document.documentElement.classList.add('dark')
+                    $dispatch('dark-mode-changed', { mode: 'dark' })
```
Comment (ryangjchandler) [inline packages/admin/resources/views/components/layouts/app/topbar/user-menu.blade.php:38]: I prefer `dark-mode-toggled` personally.

**Review by ryangjchandler — CHANGES_REQUESTED** [review]

File: packages/admin/resources/views/components/layouts/app/topbar/user-menu.blade.php (hunk 1 above)
Comment (mohamedsabil83) [reply packages/admin/resources/views/components/layouts/app/topbar/user-menu.blade.php:38]: Good advice 👌🏽

**Review by ryangjchandler — CHANGES_REQUESTED** [review]
I think we just need to fix some grammar in the docs and it'll be good to merge. Can imagine this will be handy for people who are building custom components etc. 

Will take a look at the docs tomorrow/Monday.

**Comment by danharrin** [comment]
Thanks! I've removed the event payload object and just replaced it with a `dark` or `light` string.

**Comment by mohamedsabil83** [comment]
You're welcome. Yes, that's better 👍🏽

**Pull request merged**